gigareview/frontend/views/manuscript/index.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+use common\models\Manuscript;
+use yii\helpers\Html;
+use yii\helpers\Url;
+use yii\grid\ActionColumn;
+use yii\grid\GridView;
+use yii\widgets\Pjax;
+/** @var yii\web\View $this */
+/** @var yii\data\ActiveDataProvider $dataProvider */
+
+$this->title = Yii::t('app', 'Manuscripts');
+$this->params['breadcrumbs'][] = $this->title;
+?>
+<div class="manuscript-index">
+
+    <h1><?= Html::encode($this->title) ?></h1>
+
+    <p>
+        <?= Html::a(Yii::t('app', 'Create Manuscript'), ['create'], ['class' => 'btn btn-success']) ?>
+    </p>
+
+    <?php Pjax::begin(); ?>
+
+    <?= GridView::widget([
+        'dataProvider' => $dataProvider,
+        'columns' => [
+            ['class' => 'yii\grid\SerialColumn'],
+
+            'id',
+            'doi',
+            'manuscript_number',
+            'article_title',
+            'publication_date',
+            //'editorial_status',
```

Comment:
@pli888 Is it better uncomment these to make them visible in manuscript index page?